src/twiliosim.cr

```diff
@@ -38,6 +38,6 @@ class Twiliosim::Server
   end
 end
 
-server = Twiliosim::Server.new("0.0.0.0", 8000)
+server = Twiliosim::Server.new("0.0.0.0", 80)
```

Comment:
```suggestion
server = Twiliosim::Server.new("0.0.0.0", ENV.fetch("PORT", "3000").to_i)
```

It's better to keep by default a port > 1024 for development.
If needed it can be overriden in the docker image.